app/locales/ar.ts

```diff
@@ -0,0 +1,162 @@
+import { SubmitKey } from "../store/app";
+import type { LocaleType } from "./index";
+
+const ar: LocaleType = {
+  WIP: "جاري العمل على التحديث...",
+  Error: {
+    Unauthorized:
+      "تسجيل دخول غير مصرح به، يرجى إدخال رمز الوصول في صفحة الإعدادات.",
+  },
+  ChatItem: {
+    ChatItemCount: (count: number) => `${count} رسالة`,
+  },
+  Chat: {
+    SubTitle: (count: number) => `${count} رسائل مع ChatGPT`,
+    Actions: {
+      ChatList: "الذهاب إلى قائمة المحادثات",
+      CompressedHistory: "الذاكرة العشوائية للذاكرة المضغوطة",
+      Export: "تصدير جميع الرسائل كعلامة تبويب ماركداون",
+      Copy: "نسخ",
+      Stop: "إيقاف",
+      Retry: "إعادة المحاولة",
+    },
+    Rename: "إعادة تسمية الدردشة",
+    Typing: "الكتابة...",
+    Input: (submitKey: string) => {
+      var inputHints = `اكتب شيئًا واضغط على ${submitKey} للإرسال`;
+      if (submitKey === String(SubmitKey.Enter)) {
+        inputHints += ", اضغط Shift + Enter للتالي";
+      }
+      return inputHints;
+    },
+    Send: "إرسال",
+  },
+  Export: {
+    Title: "جميع الرسائل",
+    Copy: "نسخ الكل",
+    Download: "تحميل",
+    MessageFromYou: "الرسالة منك",
+    MessageFromChatGPT: "رسالة من ChatGPT",
+  },
+  Memory: {
+    Title: "التذكير الذهني",
+    EmptyContent: "لا يوجد شيء حتى الآن.",
+    Copy: "نسخ الكل",
+  },
+  Home: {
+    NewChat: "دردشة جديدة",
+    DeleteChat: "تأكيد حذف المحادثة المحددة؟",
+  },
+  Settings: {
+    Title: "الإعدادات",
+    SubTitle: "كافة الإعدادات",
+    Actions: {
+      ClearAll: "مسح كافة البيانات",
+      ResetAll: "إعادة تعيين كافة الإعدادات",
+      Close: "إغلاق",
+    },
+    Lang: {
+      Name: "اللغة",
```

Comment:
Please set the name here to "Language". The reason behind this approach is to prevent potential confusion when changing the application language, as users may accidentally select a completely unfamiliar language to them and then struggle to revert the settings.

In order to maintain consistency across all languages, a separate pull request (#532) has also been submitted to modify the current language setting for the Italian language, where "Lingue" will be replaced with "Language." This approach has been applied to other available languages, such as Spanish, Chinese, and English as well, ensuring a user-friendly and easily navigable experience regardless of the chosen language.

Reply:
Thank you for explaining that to me. I understand now and will make the necessary adjustments.